hw1/02-palindrome.html

```diff
@@ -6,20 +6,21 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
```

Comment:
closing angular brace can be put in the previous line . code beautification.

Reply:
This was probably defaulted by the Prettier VS code extension to prevent the line from being too long.